VRDR.CLI/Program.cs

```diff
@@ -668,6 +668,67 @@ static int Main(string[] args)
                     Console.WriteLine($"{info.Field, -5} {info.Name,-15} {Truncate(info.Contents, 75), -75}: \"{field + "\"",-80}");
                 }
             }
+            else if (args.Length == 2 && args[0] == "extract")
+            {
+                BaseMessage message = BaseMessage.Parse(File.ReadAllText(args[1]));
+                switch(message)
+                {
+                    case DeathRecordSubmission submission:

```

Comment:
Does this also work for DeathRecordUpdate instances (a subclass of DeathRecordSubmission)?

Reply:
It does!